Sources/Foundation/Documentation.docc/Curation/FoundationEssentials.md

```diff
@@ -0,0 +1,37 @@
+# Foundation Essentials
+
+Access core types and essential functionality used by most applications.
+
+## Overview
+
+The Foundation Essentials package provides the most important types for everyday development:
+
+* Core types beyond those provided by the Swift standard library, such as ``Data``, ``Decimal``, and ``UUID``.
```

Comment:
Alas, we don't have `Decimal` or a concrete plan of what to do about it. Its fate is TBD. Do we still want it here?